bundles/GlossaryBundle/public/js/glossary.js

```diff
@@ -57,7 +57,7 @@ pimcore.settings.glossary = Class.create({
 
         var itemsPerPage = pimcore.helpers.grid.getDefaultPageSize();
         this.store = pimcore.helpers.grid.buildDefaultStore(
-            Routing.generate('pimcore_admin_settings_glossary'),
+            Routing.generate('pimcore_glossary_settings_glossary'),
```

Comment:
maybe we should move that to the `pimcore.glossary.*` namespace as well, to be consistent.

Reply:
Well strange, this is already in the glossary namespace in my file. :) 
Otherwise the routing would not work because I changed the route in the Settingscontroller too.